src/containers/Preview/ProjectPreview/index.js

```diff
@@ -139,7 +142,7 @@ function ProjectPreview(props) {
                       <div>{currentProject.name}</div>
 
                       <div className="configuration">
-                        <Link to="/" className="go-back-button-preview">
+                        <Link onClick={() => history.go(-1)} className="go-back-button-preview">
```

Comment:
If it is for go back, use goBack function of history

Reply:
changed